recipes/diffpy.pdfgui/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "diffpy.pdfgui" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/diffpy/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4cf3106323544bbd6ead3a774c4dd1db534c618c116e0a0811a33493371150f5
+
+build:
+  # we need egg_dir to avoid overlapping diffpy/__init__.py files.
+  preserve_egg_dir: True
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  entry_points:
+    - pdfgui = diffpy.pdfgui.applications.pdfgui:main
+  osx_is_app: True
+
+requirements:
+  host:
+    - python =3.7
+    - pip
+    - setuptools
+
+  run:
+    - python =3.7
```

Comment:
```suggestion
    - python >=3.6
```

Reply:
Is this package really 3.7 only? The metadata says it is `>=3.6`. If that is true you can remove the constraint.

Reply:
It is python>=3.6. But since we are removing `noarch`, the conda-forge-linter warns that  "Non noarch packages should have python requirement without any version constraints.". Shall we ignore this error?